cmd/crc/cmd/daemon.go

```diff
@@ -243,6 +254,12 @@ func gatewayAPIMux() *http.ServeMux {
 	return mux
 }
 
+func networkAPIMux(vn *virtualnetwork.VirtualNetwork) *http.ServeMux {
+	mux := http.NewServeMux()
+	mux.Handle("/", vn.Mux())
+	return mux
+}
+
```

Comment:
Is `host.crc.testing:7777` hardcoded? Or would it have been possible to go through `gateway` instead by reusing/extending `gatewayAPIMux()`?

Reply:
This will change anyway with Podman 4.x, right?
I do not think we need to spend a lot of time on this ATM.

Reply:
Did I ask for changes? This is only to improve my understanding of this, in case Praveen knows this off-hand.